modules/ml/src/rtrees.cpp

```diff
@@ -422,6 +422,18 @@ Ptr<RTrees> RTrees::create()
     return makePtr<RTreesImpl>();
 }
 
+//Function needed for Python and Java wrappers
+Ptr<RTrees> RTrees::load(const String& filepath)
+{
+    FileStorage fs;
+    fs.open(filepath, FileStorage::READ);
+
+    Ptr<RTrees> rtree = makePtr<RTreesImpl>();
+
+    ((RTreesImpl*)rtree.get())->read(fs.getFirstTopLevelNode());
```

Comment:
@sovrasov : fs.root() doesn't seem to work. I trained an RTree and stored it in YAML format. Later I tried to use the method you suggested to load the RTree. I'm attaching "RTrees_test.dat" and the "test.cpp" files I used to check this. 
The tree loads fine but is unable to predict. @berak suggested that this is becuase the fs.root() method selects the wrong node.
[test.zip](https://github.com/opencv/opencv/files/710955/test.zip)